plugins/modules/filesystem.py

```diff
@@ -275,7 +276,7 @@ def grow(self, dev):
             self.module.warn(f"unable to process {self.INFO} output '{err}'")
             self.module.fail_json(msg=f"unable to process {self.INFO} output for {dev}")
 
-        if not fssize_in_bytes < devsize_in_bytes:
+        if not fssize_in_bytes + self.GROW_SLACK < devsize_in_bytes:
```

Comment:
I think this is more readable with brackets:
```suggestion
        if not (fssize_in_bytes + self.GROW_SLACK < devsize_in_bytes):
```

Reply:
I definitely cast my vote on this last one: ditch the `not`, just invert the comparison, no brackets required